lib/use_cases/performance_platform/publish_locations_ips.rb

```diff
@@ -0,0 +1,15 @@
+class PublishLocationsIps
+  def initialize(upload_gateway:, ips_gateway:)
```

Comment:
We called these `destination gateway` and `source_gateway` on the other half of this work, and I like that better than calling either of these `ip_gateway`, as it might imply they're not dealing with the same data.

Reply:
makes sense to have them the same